botchat.css

```diff
@@ -122,6 +116,11 @@
         position: relative;
     }
 
+    .wc-message-wrapper.carousel .wc-message {
```

Comment:
but your code is generating a space between `.wc-message-wrapper` and `.carousel`